src/Symfony/Component/Notifier/Exception/MultipleExclusiveOptionsUsedException.php

```diff
@@ -17,15 +17,16 @@
 class MultipleExclusiveOptionsUsedException extends InvalidArgumentException
 {
     /**
-     * @param string[]      $usedExclusiveOptions
-     * @param string[]|null $exclusiveOptions
+     * @param string[] $usedOptions
+     * @param string[] $exclusiveOptions
      */
-    public function __construct(array $usedExclusiveOptions, array $exclusiveOptions = null, \Throwable $previous = null)
+    public function __construct(array $usedOptions, array $exclusiveOptions, \Throwable $previous = null)
```

Comment:
I don't think we should rename the argument. The code might have used other options that were not part of the exclusive options and those should not be passed to this argument.